go/base/static/js/src/components/views.js

```diff
@@ -42,10 +42,30 @@
           $p = $('<p class="textarea-char-count"/>');
           this.$el.after($p);
       }
-      this.totalChars = this.$el.val().length;
-      this.totalSMS = Math.ceil(this.totalChars / this.SMS_MAX_CHARS);
-      $p.html(this.totalChars + ' characters used<br>' + this.totalSMS + ' smses');
-
+      var text = this.$el.val();
+      var non_ascii = go.utils.non_ascii(text);
+      this.containsNonAscii = (non_ascii.length > 0);
+      this.totalChars = text.length;
+      this.totalBytes = text.length * (1 + this.containsNonAscii);
+      this.totalSMS = Math.ceil(this.totalBytes / this.SMS_MAX_CHARS);
+      var html;
+      if (this.containsNonAscii) {
+          html = [
+              'Non-ASCII characters: ' + non_ascii.join(', '),
+              this.totalChars + ' characters used (~' +
+                  this.totalBytes + ' bytes)',
```

Comment:
I know this is getting very bikesheddy, but I'm wondering how useful showing the bytes is to people. I doubt most people will understand the implications of the bytes. I vote we either don't display the byte count, or we mention the implications of the byte count.